introduction.md

```diff
@@ -1,5 +1,6 @@
 I love Ruby
 
+* Java
 * Ruby
 * JavaScript
-* rebase
\ No newline at end of file
+* rebase
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?